src/app/auth/auth.component.html

```diff
@@ -0,0 +1,35 @@
+<app-header-static>
+</app-header-static>
+<div class="container-top">
+  <div class="auth-container" style="background-image: url('assets/images/auth_background.png');">
+    <div class="auth-background-image" style="background-image: url('assets/images/auth_orange_image.png');">
+      <span class="auth-tagline">Evaluating state of the art in AI</span>
+    </div>
+    <div class="auth-form">
+      <div class="auth-form-logo">
+        <img src="assets/images/evalai_logo_light.png" />
+      </div>
+      <div class="auth-nav">
+        <a [class.auth-selected]="localRouter.url=='/auth/login'"  (click)="navigateTo('/auth/login')">LOGIN</a>
```

Comment:
Adding click to anchor tag is not a good Practice instead use routerLink for the navigation @live-wire

Reply:
Use like this `<a [routerLink]="['URL']" routerLinkActive="auth-selected">` @live-wire